src/main/proto/ga4gh/variants.proto

```diff
@@ -191,4 +191,29 @@ message Variant {
   // determination of genotype with respect to this variant. `Call`s in this
   // array are implicitly associated with this `Variant`.
   repeated Call calls = 12;
+
+  // The "variant_type" is used to denote e.g. structural variants.
+  // Examples:
+  //   DUP  : duplication of sequence following "start"; not necessarily in situ
+  //   DEL  : deletion of sequence following "start"
+  string variant_type = 13;
```

Comment:
You mean you want to use a controlled vocabulary when it's available, and an open string otherwise? In that case we can make the field a oneof a string or ontology term. This makes constructing the message a little more baroque, but captures the cases nicely.